.travis.yml

```diff
@@ -26,13 +31,15 @@ before_install:
 
 install:
     - pip install mkdocs mkdocs-material
+    - curl -sL -o ${HADOLINT} "https://github.com/hadolint/hadolint/releases/download/v1.16.0/hadolint-$(uname -s)-$(uname -m)" && chmod 700 ${HADOLINT}
```

Comment:
Ideally we'd run all the CI steps in the docker build so that we don't get into the situation of having parts of the build difficult to run outside of the CI server. Please move the hadolint install and invocation to the Dockerfile's builder stage.